terraform_files/main.tf

```diff
@@ -132,3 +140,27 @@ resource "libvirt_domain" "worker" {
     dev = ["hd", "cdrom"]
   }
 }
+
+data "libvirt_network_dns_host_template" "api" {
```

Comment:
lets add nice comment for future use , how it works. I think we will have more stuff like this and be great that we will have explanation for this beautiful code

Reply:
done